solutions/09-tradeoffs/movr/cockroach/init_db.sh

```diff
@@ -23,6 +23,11 @@ echo "CONNECTING TO COCKROACH_DB"
 echo "$URL"
 echo "----------------------------------------------------------------------"
 
+echo "----------------------------------------------------------------------"
+echo "UPDATING CLUSTER TO SUPPORT PLACEMENT RESTRICTED "
+echo "----------------------------------------------------------------------"
+cockroach sql --url $URL --execute 'SET CLUSTER SETTING sql.defaults.multiregion_placement_policy.enabled = on;'
+
```

Comment:
I see this on problems 8 & 9 but not before. I assume that's correct but wanted to call it out & confirm.

Reply:
That's correct. It's only for 8 & 9